### PR TITLE
Deprecate the script

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17654

## Description
A few sentences describing the overall goals of the pull request's commits.

## Does it break backward compatibility?
   - No

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [x] CBFindIP
- [ ] Dependency 2
- [ ] Dependency 3

### CODE REVIEW

**Comment by Itay4** [comment]
@kirbles19 we are deprecating this script
would you like to edit the script description?
@Shellyber please comment here what commands to use instead of this script

**Comment by Shellyber** [comment]
Instead of the script, use the commands from Carbon Black Response:
!cb-binary
!cb-get-processes

**Comment by kirbles19** [comment]
@Shellyber  Updated RN and script description.

File: Scripts/CBFindIP/CBFindIP.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        else:
+            found.append(ip)
+            t += e['HumanReadable']
+if t:
+    appendContext("found_ips", ','.join(found), dedup=True)
+    demisto.results({'ContentsFormat': formats['markdown'], 'Type': entryTypes['note'], 'Contents': e['HumanReadable'],
+                     'EntryContext': e['EntryContext']})
+else:
+    demisto.results({'ContentsFormat': formats['text'], 'Type': entryTypes['note'], 'Contents': 'No results.'})
+demisto.results(res)
```
Comment (Itay4) [inline Scripts/CBFindIP/CBFindIP.py]: this is not needed, lets just `return_error` if `isError`

**Comment by Itay4** [comment]
@kirbles19 please also review `CBFindIP.yml`

**Pull request merged**